From fd12952d71f36ca833b34723982d5ede04149893 Mon Sep 17 00:00:00 2001
From: Stanislaw Kardach <skardach@marvell.com>
Date: Mon, 10 Feb 2020 21:46:24 +0100
Subject: [PATCH 475/767] octeontx2-rm: fix compilation warning

commit 91a852adac0c245214d0e8ba20ea265db4c59e2e from
git@git.assembla.com:cavium/WindRiver.linux.git

Remove unused variables.

Change-Id: I33cd922552980605a6e8c95e652c51018d28ccaf
Signed-off-by: Stanislaw Kardach <skardach@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/23184
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stanislaw Kardach <Stanislaw.Kardach@cavium.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2-rm/domain_sysfs.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/soc/marvell/octeontx2-rm/domain_sysfs.c b/drivers/soc/marvell/octeontx2-rm/domain_sysfs.c
index 9101edea8118..6083db40d0ed 100644
--- a/drivers/soc/marvell/octeontx2-rm/domain_sysfs.c
+++ b/drivers/soc/marvell/octeontx2-rm/domain_sysfs.c
@@ -685,10 +685,8 @@ enadis_pmccntr_el0_store(struct kobject *kobj, struct kobj_attribute *attr,
 	char tmp_buf[64];
 	long enable = 0;
 	char *tmp_ptr;
-	ssize_t used;
 
 	strlcpy(tmp_buf, buf, 64);
-	used = strlen(tmp_buf);
 	tmp_ptr = strim(tmp_buf);
 	if (kstrtol(tmp_ptr, 0, &enable)) {
 		dev_err(dev, "Invalid value, expected 1/0\n");
-- 
2.31.1

